### PR TITLE
Initial Chromatic setup

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

Initial setup of Chromatic: [LG-2910](https://jira.mongodb.org/browse/LG-2910)

This does _not_ set up any CI,  play-function, or add any additional tests.

More updates to come.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,13 +28,15 @@
     "build": "npm-run-all --serial \"build:* {@}\" --",
     "build:packages": "yarn ts-node ./scripts/build.ts",
     "build:ts": "yarn ts-node ./scripts/ts-build.ts",
+    "build-storybook": "build-storybook",
```
Comment (bruugey) [inline package.json:31]: won't this work by default? do we need to write it explicitly in our package.json?

File: packages/banner/src/Banner.story.tsx [hunk 2]
```diff
@@ -84,3 +84,6 @@ export const WithCustomImage: ComponentStory<typeof Banner> = ({ ...args }) => (
 WithCustomImage.args = {
   variant: Variant.Info,
 };
+WithCustomImage.parameters = {
```
[the remark this answers is not in the document]
Comment (bruugey) [reply packages/banner/src/Banner.story.tsx:87]: ditto to all other .story files

File: package.json (hunk 1 above)
Comment (TheSonOfThomp) [reply package.json:31]: The chromatic script tries to run the storybook 7.x script `storybook dev` by default. We can pass in a custom yarn script though, so we need to explicitly define it like this till we upgrade to SB 7.x

File: packages/banner/src/Banner.story.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (TheSonOfThomp) [reply packages/banner/src/Banner.story.tsx:87]: The ones I changed in this PR were causing errors. The rest of them will come later

File: packages/hooks/src/useAutoScroll/useAutoScroll.story.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ const testItems = 16;
 
 export default StoryMeta({
   title: 'hooks/useAutoScroll',
+  component: undefined,
```
Comment (bruugey) [inline packages/hooks/src/useAutoScroll/useAutoScroll.story.tsx:15]: did we double check that the effected stories are still rendering properly on .design?

Comment (TheSonOfThomp) [reply packages/hooks/src/useAutoScroll/useAutoScroll.story.tsx:15]: None of the updated ones here are being used on .design

File: packages/banner/src/Banner.story.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (bruugey) [reply packages/banner/src/Banner.story.tsx:87]: Banner is on .design - double check this one

File: stories/forms/SampleForm.stories.tsx [hunk 4]
```diff
@@ -5,6 +5,9 @@ import TextInput from '@leafygreen-ui/text-input';
 
 export default {
   title: 'Form/Sample Forms',
+  parameters: {
```
Comment (bruugey) [inline stories/forms/SampleForm.stories.tsx:8]: do we even need this file?

File: stories/sample-pages/layout-1.stories.jsx [hunk 5]
```diff
@@ -3,6 +3,9 @@ import Button from '@leafygreen-ui/button';
 
 export default {
   title: 'Sample Pages (WIP)/Layout 1',
+  parameters: {
+    chromatic: { disableSnapshot: true },
```
Comment (bruugey) [inline stories/sample-pages/layout-1.stories.jsx:7]: ditto here

Comment (TheSonOfThomp) [reply stories/sample-pages/layout-1.stories.jsx:7]: We only use component stories on the website.

Comment (bruugey) [reply stories/sample-pages/layout-1.stories.jsx:7]: Just Banner - I was saying we could rm these two files

File: stories/forms/SampleForm.stories.tsx (hunk 4 above)
Comment (TheSonOfThomp) [reply stories/forms/SampleForm.stories.tsx:8]: Not sure. Sean created it. I think it's a wip

**Review by bruugey — APPROVED** [review]

**Review by bruugey — APPROVED** [review]

**Pull request merged**